### PR TITLE
Add tests for config options

### PR DESCRIPTION
**Pull request by st0012**
- `skip_path`
- `show_src_lines`
- `show_frames`

### CODE REVIEW

File: test/debug/config_test.rb [hunk 1]
```diff
@@ -67,4 +67,142 @@ def test_config_show_frames_set
...
+        end
+        def lib_m2
+          2
+        end
+      RUBY
+    end
+
+    def program(lib_file)
+      <<~RUBY
+     1| DEBUGGER__::CONFIG[:skip_path] = [/library/]
```
Comment (st0012) [inline test/debug/config_test.rb:159]: I imagine this config will more likely to be set with code instead of command.

Comment (ko1) [reply test/debug/config_test.rb:159]: Do not modify `CONFIG` hash because they will be frozen for Ractor support. Use `set_config skip_path: [...]`.

Comment (ko1) [reply test/debug/config_test.rb:159]: Sorry I will prepare `CONFIG#[]=(key, val)` to change the configuration, so no need to change.

Comment (st0012) [reply test/debug/config_test.rb:159]: Thanks for the update. I think this PR is ready for merge if no other changes are needed?

**Pull request merged**